plugins/node/opentelemetry-instrumentation-ioredis/README.md

```diff
@@ -49,6 +49,7 @@ IORedis instrumentation has few options available to choose from. You can set th
 | Options | Type | Description |
 | ------- | ---- | ----------- |
 | `dbStatementSerializer` | `DbStatementSerializer` | IORedis instrumentation will serialize db.statement using the specified function. |
+| `requestHook` | `RedisRequestCustomAttributeFunction` | Function for adding custom attributes on db request |
```

Comment:
Maybe add some more info in readme how this hook is used and which data is given to it.

Reply:
@Flarna added the function signature to README.
I'm open to suggestions on how to improve it further.

Reply:
A short sample like for `dbStatementSerializer` could be added. I assume you have some use of this already somewhere in your samples/tests/... you could add there.

Reply:
added requestHook example to README